### PR TITLE
Metadata/InstallerType: AutoIt.AutoIt version 3.3.14.5

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22309)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@denelon @ChuckFerring Please check why old commits are being added to new PRs.

File: manifests/a/AutoIt/AutoIt/3.3.14.5/AutoIt.AutoIt.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageVersion: 3.3.14.5
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
```
Comment (ItzLevvie) [inline manifests/a/AutoIt/AutoIt/3.3.14.5/AutoIt.AutoIt.installer.yaml:13]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
> @denelon @ChuckFerring Please check why old commits are being added to new PRs.

That's a question for https://github.com/microsoft/winget-pkgs/pull/21204.

**Comment by vedantmgoyal2009** [comment]
You mean that's because of that PR. But how?

**Comment by ItzLevvie** [comment]
I believe it's related to these two lines: https://github.com/microsoft/winget-pkgs/pull/21204/files#diff-4fad6db3631f6d12d20c4746d1a833deddd3b62d3c5e6d47ff36a8e56ecedd75R817-R818

**Comment by vedantmgoyal2009** [comment]
So, he should write master in that place.

**Review by ItzLevvie — APPROVED** [review]

**Comment by OfficialEsco** [comment]
Since you are using the script without advertising that you are using the script (stealing) this seems like a problem you can fix your self.
Its fixed, just not in that PR.

**Comment by vedantmgoyal2009** [comment]
Yes, I fixed it on my machine. No issues now.

**Pull request merged**